wallet/udb/agendas.go

```diff
@@ -7,48 +7,116 @@ package udb
 import (
 	"decred.org/dcrwallet/errors"
 	"decred.org/dcrwallet/wallet/walletdb"
+	"github.com/decred/dcrd/chaincfg/chainhash"
 )
 
 type agendaPreferencesTy struct {
 }
 
 var agendaPreferences agendaPreferencesTy
 
-var agendaPreferencesRootBucketKey = []byte("agendaprefs")
-
-func (agendaPreferencesTy) rootBucketKey() []byte { return agendaPreferencesRootBucketKey }
+var (
+	defaultAgendaPrefsBucketKey = []byte("agendaprefs")
+	ticketsAgendaPrefsBucketKey = []byte("ticketsagendaprefs")
+)
 
-func (agendaPreferencesTy) key(version uint32, agendaID string) []byte {
+func prefKey(version uint32, agendaID string) []byte {
```

Comment:
why make this a non-method while still keeping the type it was a method for?

(i suggest not changing this, fwiw. the empty struct type was there to provide some sort of "namespacing" for this method).